docs/advanced/pcidevices.md

```diff
@@ -22,6 +22,8 @@ To use the PCI devices feature, users need to enable the `pcidevices-controller`
 
 ![](/img/v1.2/vm-import-controller/EnableAddon.png)
 
+Once the `pcidevices-controller` addon is deployed successfully, it can take a few minutes for it to scan and the PCIDevice CRDs to become available.
```

Comment:
```suggestion
Once the `pcidevices-controller` addon is deployed successfully, it can take a few minutes for it to scan and the PCI device CRDs to become available.
```

Should PCIDevice be replaced by PCI device?

Reply:
i should make it `PCIDevice` only to be consistent.